packages/terra-signature/src/Signature.jsx

```diff
@@ -3,18 +3,23 @@ import PropTypes from 'prop-types';
 import 'terra-base/lib/baseStyles';
 import styles from './Signature.scss';
 
-const LineWidths = {
+const LINEWIDTHS = {
   EXTRA_FINE: 1,
   FINE: 2,
   MEDIUM: 4,
   HEAVY: 6,
 };
+
+const MOUSECODES = {
+  LEFTCLICK: 0,
+};
+
 const propTypes = {
   /**
    * The line width to use when drawing the signature on the canvas.
-   * One of LineWidths.EXTRA_FINE, LineWidths.FINE, LineWidths.MEDIUM, LineWidths.HEAVY.
+   * One of Signature.Opts.Width.EXTRA_FINE, Signature.Opts.Width.FINE, Signature.Opts.Width.MEDIUM, Signature.Opts.Width.HEAVY.
```

Comment:
reason why the comment here is not using the LINEWIDTHS const that the code is using?  should the code be using Signature.Opts?

Reply:
Singature.Opts are the options obj we export to the consumer. [See here](https://github.com/cerner/terra-core/pull/929/files/cf952e026eef88ead8a22f24cd40683bb8775b11#diff-439e726998c8c3153688d80b4864792dR229)